_posts/en/newsletters/2021-07-28-newsletter.md

```diff
@@ -67,7 +67,8 @@ BOLTs][bolts repo].*
   higher than the current rate at which a Bitcoin Core node announces
   addresses.
 
-- [C-Lightning #4669][] Updates to offers FIXME:dongcarl
+- [C-Lightning #4669][] fixes several bugs in its [LN offers][topic offers]
+  parsing and validation logic reported by Nadav Igvi.
```

Comment:
We generally don't attribute individuals in the "notable code changes" section, since the only fair thing to do would be to attribute all authors, reviewers, testers, etc.

```suggestion
  parsing and validation logic.
```